components/expenses/MarkExpenseAsUnpaidBtn.js

```diff
@@ -54,8 +54,12 @@ const MarkExpenseAsUnpaidBtn = ({ id, markExpenseAsUnpaid, refetch }) => {
           </StyledButton>
         </Fragment>
       ) : (
-        <StyledButton onClick={() => setState({ ...state, showProcessorFeeConfirmation: true })} mt={2}>
-          <FormattedMessage id="expense.markAsUnpaid.btn" defaultMessage="Mark as unpaid" />
+        <StyledButton
+          buttonStyle="standard"
+          onClick={() => setState({ ...state, showProcessorFeeConfirmation: true })}
+          mt={2}
+        >
+          <FormattedMessage id="expense.markAsUnpaid.btn" defaultMessage="Mark As Unpaid" />
```

Comment:
My example in https://github.com/opencollective/opencollective/issues/2570 (`Mark as paid`) was using sentence case, sorry for confusing it by saying title case

```suggestion
          <FormattedMessage id="expense.markAsUnpaid.btn" defaultMessage="Mark as unpaid" />
```